### PR TITLE
Created nuxt category & added nuxt3-appwrite line

### PR DESCRIPTION
**Pull request by YsarocK**
## What does this PR do?

This Pull Request creates the Nuxt category, allowing navigation. It adds a line refering to the **nuxt3-appwrite** package which simplify the Appwrite usage with Nuxt 3. The package provides a `useAppwrite` composable, an easy setup & route guards based on user session (client & server).

The goal is to improve the Nuxt 3 compatibility, the current **nuxt-appwrite** package was made for Nuxt 2.

## Test Plan

Only the README file has been changed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -364,12 +363,15 @@ Share your apps here! Submit a pull request!
 
 * **Tailwinder** This an open-source complete community website project created for Appwrite hackathon. [Source Code](https://github.com/MooseSaeed/Tailwinder) [Demo](https://dev.to/moose_said/introducing-tailwinder-home-for-tailwind-fans-powered-with-appwrite-4c0o) [Video Demo](https://vimeo.com/manage/videos/709162501)
 
+* **Appwrite + Vue Podcast Transcript** A simple Podcast Transcript app built with Appwrite, Deepgram and Vue. [Source Code](https://github.com/timothee-durand/appwrite-transcript)
+
+### Nuxt
+* **[nuxt3-appwrite package](https://github.com/YsarocK/nuxt-appwrite)** A package to simply implement Appwrite in your Nuxt 3 project. Composable, Routes guards & no config needed. [Documentation](https://nuxt-appwrite.etiennemoureton.fr)
```
Comment (adityaoberai) [inline README.md:369]:
```suggestion
* **nuxt3-appwrite package** A package to simply implement Appwrite in your Nuxt 3 project. Composable, Routes guards & no config needed. [Source Code](https://github.com/YsarocK/nuxt-appwrite), [Documentation](https://nuxt-appwrite.etiennemoureton.fr)
```

Suggesting this change to ensure consistency of format with other submissions in the file

**Review by adityaoberai — CHANGES_REQUESTED** [review]
@YsarocK thank you for contributing the package. 

For now, however, we would really appreciate if you could revert the separation of the Nuxt category. The main reason for this is that we have not maintained separations for other such frameworks, like Next for React or SvelteKit for Svelte. I am currently performing a cleanup of the repo to archive outdated projects and content as well, and will re-evaluate this in the process.

That being said, we'd be happy to accept the package as a contribution (with the suggested edit)

**Comment by YsarocK** [comment]
Hey! Thanks for the review. I don't see any problem about reverting the section creation. I handle this today & get you in touch. Thanks!

**Comment by adityaoberai** [comment]
> Hey! Thanks for the review. I don't see any problem about reverting the section creation. I handle this today & get you in touch. Thanks!

Thank you! Once you make the changes, please feel free to re-request a review and I'll look through it.

**Review by adityaoberai — APPROVED** [review]
Looks good to me 👍

**Pull request merged**